src/main/java/com/juniorstart/juniorstart/controller/UserController.java

```diff
@@ -24,17 +26,23 @@
 @RestController
 public class UserController {
     final private UserService userService;
-
+    final private UserDao userDao;
```

Comment:
O nie nie, nie ma mowy :P UserDao musi stąd wylecieć :) Kontroler może komunikować się tylko z serwisami.

Reply:
to samo co na górze